### PR TITLE
FORMS-8863 datalayer clientlib in Sites Page and xfpage if forms incl…

### PR DESCRIPTION
**Pull request by barshat7**
…uded and datalayer included

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by ujjwal5 — APPROVED** [review]
LGTM

**Review by ujjwal5 — APPROVED** [review]

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html [hunk 1]
```diff
@@ -35,5 +35,9 @@
      data-sly-test.hasForm="${symbol_dollar}{formstructparser.containsFormContainer}">
     <sly data-sly-test="${symbol_dollar}{hasForm}" data-sly-call="${symbol_dollar}{clientlib.js @ categories='core.forms.components.runtime.all', async=true}"/>
 </sly>
+#if ( $datalayer == "y")
```
Comment (ujjwal5) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html:38]: Please make sure that the clientlib inclusion has to be similar to this, [https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main[…]root/apps/core/fd/components/page/v1/page/customfooterlibs.html](https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v1/page/customfooterlibs.html)

Comment (barshat7) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html:38]: its the same

**Review by ujjwal5 — APPROVED** [review]

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html (hunk 1 above)
Comment (rismehta) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html:38]: @barshat7 Please write a comment explicitly referring the original file from where we have copied the content

**Review by rismehta — APPROVED** [review]

**Pull request merged**